app/controllers/stocks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class StocksController < ApplicationController
+  def index
+    stocks = Stock.all
+    render json: StockSerializer.new(stocks).serializable_hash, status: 200
+  end
+
+  def show
+    stock = Stock.find(params[:id])
+
+    render json: StockSerializer.new(stock).serializable_hash, status: 200
+  end
+
+  # def create
+  #   stock = Stock.new(stock_params)
+  #
+  #   if stock.save
+  #     render json: StockSerializer.new(stock).serializable_hash, status: 201
+  #   else
+  #     render json: stock.errors.details
+  #   end
+  # end
+
+  # def update
+  #   stock = Stock.find(params[:id])
+  #
+  #   if stock.update(stock_params)
+  #     render json: StockSerializer.new(stock).serializable_hash, status: 200
+  #   else
+  #     render json: stock.errors.details
+  #   end
+  # end
```

Comment:
we can remove these comments, I don't think we'll give controls to users to change our stocks table